app/views/UserNotificationPreferencesView/index.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import {
+	View, ScrollView, Text
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+import { themes } from '../../constants/colors';
+import StatusBar from '../../containers/StatusBar';
+import ListItem from '../../containers/ListItem';
+import Separator from '../../containers/Separator';
+import I18n from '../../i18n';
+import scrollPersistTaps from '../../utils/scrollPersistTaps';
+import styles from './styles';
+import RocketChat from '../../lib/rocketchat';
+import { withTheme } from '../../theme';
+import SafeAreaView from '../../containers/SafeAreaView';
+import SectionTitle from '../NotificationPreferencesView/SectionTitle';
```

Comment:
I know we have to create a UI lib asap, but IMO adds more coupling and complexity to the code.
Just duplicate instead of import from another view.
We're going to move these components away from these views soon 🤞